README.md

```diff
@@ -97,6 +97,24 @@ docker run -p 9090:9090 prom/prometheus --config.file=/etc/prometheus/prometheus
              --web.enable-remote-write-receiver
 ```
 
+## Enable tracing in Docker
+
+QuickPizza will automatically push traces in OTLP format to the URL specified in `QUICKPIZZA_OTLP_ENDPOINT`, if any. A common pattern to forward traces to your tracing backend of choice (e.g. Grafana Tempo, Grafana Cloud) is by running an Opentelemetry collector container as a forwarder. We can easily set this up with the following commands:
+
+```shell
+# Create a docker network so both containers can talk to each other:
+docker network create quickpizza
+# Configure URL, user, and API key for your tracing backend:
+# For Grafana cloud, see https://grafana.com/docs/opentelemetry/collector/send-otlp-to-grafana-cloud-databases/
+ export TRACES_ENDPOINT=your-tracing-endpoint.grafana.net:443
+ export TEMPO_USER=12345
+ export TEMPO_API_KEY=changeme
+# Run the opentelemetry collector. The Kubernetes config for the collector works well in Docker as well.
+docker run --name otelcol --rm -i -v ./kubernetes/otelcol/config/otelcol.yaml:/etc/otelcol-contrib/config.yaml --network quickpizza -e TRACES_ENDPOINT -e TEMPO_USER -e TEMPO_API_KEY otel/opentelemetry-collector-contrib
```

Comment:
I defaulted to otelcol merely because I'm more familiar with it. I suspect there will also be more people familiar with it than with the Grafana agent. Do we have a strong reason to prefer the latter? (e.g. it being simpler to setup somehow)

Reply:
The Grafana Agent embeds the OTEL Collector, so the tracing config should be mostly the same.

IMO: Easier to setup for Prometheus metrics (Mimir) and Logs (Loki). Also, it is used in most Grafana Labs demos.

Reply:
Gave the grafana agent a swing here: https://github.com/grafana/quickpizza/pull/12/commits/a6f99dde0894ee2f7eb98066d13b2211140c105c

For now only tracing is enabled. Enabling prometheus metrics should be easy but I think we can leave that for a follow-up PR.